Kysect.BotFramework/Core/BotMedia/IBotOnlineFile.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Kysect.BotFramework.Core.BotMedia
+{
+    public interface IBotOnlineFile : IBotMediaFile
+    {
+        public string Id { get; }
```

Comment:
~~public~~

Reply:
> ~public~

```suggestion
        string Id { get; }
```

Reply:
```suggestion
        string Id { get; }
```

Ну тоже так умею и чо